packages/utils/src/EventHandlers/DerivedEventHandlers.ts

```diff
@@ -0,0 +1,39 @@
+import { EventHandlers } from './EventHandlers';
+import { EventHandlerConnectors, EventHandlerUpdates } from './interfaces';
+
+export abstract class DerivedEventHandlers<
```

Comment:
Note to self: what do derived handlers do ? 🤔

Reply:
It's a way to create EventHandlers that depend on another EventHandler. The derived EventHandlers would have be able to easily create connectors that composites of the parents connectors. Additionally, when the parent handlers are disable/enabled, so will the derived handlers

This is the case with the LayerEventHandlers which depends on the CoreEventHandlers.